lib/manager/bazel/extract.js

```diff
@@ -139,6 +139,7 @@ function extractPackageFile(content) {
       const parsedUrl = parseUrl(url);
       dep.depName = depName;
       dep.repo = parsedUrl.repo;
+      dep.currentDigest = sha256;
```

Comment:
To work with the existing process/lookup, dep.currentDigest should actually be the git *commit*, not the sha256